src/main/java/com/google/api/codegen/config/PageStreamingConfigGenerator.java

```diff
@@ -54,15 +56,16 @@
 
   private Map<String, Object> generateRequestConfig(Method method) {
     MessageType inputMessage = method.getInputMessage();
+    Map<String, Object> requestConfig = new LinkedHashMap<String, Object>();
     for (Field field : inputMessage.getFields()) {
       String fieldName = field.getSimpleName();
       if (fieldName.equals(PARAMETER_PAGE_TOKEN)) {
-        Map<String, Object> requestConfig = new LinkedHashMap<String, Object>();
         requestConfig.put(CONFIG_KEY_TOKEN_FIELD, fieldName);
-        return requestConfig;
+      } else if (fieldName.equals(PARAMETER_PAGE_SIZE)) {
+        requestConfig.put(CONFIG_KEY_PAGE_SIZE_FIELD, fieldName);
       }
     }
-    return null;
+    return requestConfig;
```

Comment:
Either this line should return null or you need to modify the null check in L45.